src/libs/actions/App.ts

```diff
@@ -527,6 +527,10 @@ function handleRestrictedEvent(eventName: string) {
     API.write('HandleRestrictedEvent', parameters);
 }
 
+function updateLastVisitedPath(path: string) {
+    Onyx.set(ONYXKEYS.LAST_VISITED_PATH, path);
```

Comment:
```suggestion
    Onyx.merge(ONYXKEYS.LAST_VISITED_PATH, path);
```

Is there a reason to use set? If not let's always try to use merge since it's non-blocking

Reply:
This must be leftover from `WORKSPACE_ID` onyx key which was replaced by this. Now It can be replaced with the merge.